sway/commands/resize.c

```diff
@@ -62,225 +62,230 @@ static bool resize_floating(int amount, bool use_width) {
 	return false;
 }
 
-static bool resize_tiled(int amount, bool use_width) {
-	swayc_t *parent = get_focused_view(swayc_active_workspace());
-	swayc_t *focused = parent;
-	swayc_t *sibling;
-	if (!parent) {
-		return true;
+/**
+ * Return the number of children in the slave groups. This corresponds to the children
+ * that are not members of the master group.
+ */
+static inline uint_fast32_t slave_count(swayc_t *container) {
+	return container->children->length - container->nb_master;
+
+}
+
+/**
+ * given the index of a container's child, return the index of the first child of the group
+ * which index is a member of.
+ */
+static int group_start_index(swayc_t *container, int index) {
+	if ((index < 0) || (! is_auto_layout(container->layout)) ||
+	    ((uint_fast32_t) index < container->nb_master)) {
```

Comment:
trying out "spacemacs", haven't yet looked into having it format using tabs only. For now fixed it up manually.